woocommerce/wc-admin
# Future proof experimental components

Fixes #5873

Use non experimental versions of components for future-proofing but fallback to experimental if they don't exist yet.

### Detailed test instructions:

1. Check out this branch.
2. Make sure things still run as expected, especially around `Text` and navigation components.
3. If you really want to check that things continue to work, optionally spin up a dev version of GB and update named exports in  `@wordpress/components`.
